app/views/content/subjects/physics/_article.html.erb

```diff
@@ -0,0 +1,198 @@
+<div class="row inset">
+<section class="col col-720">
+  <p>
+    Teaching physics is an exciting and rewarding career. Schools need more specialist physics teachers, so it's a great choice of subject.
+  </p>
+
+  <p>
+    If you train to teach physics, you could also be eligible receive a scholarship or bursary of up to £29,000.
+  </p>
+  </section>
+
+   <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "Spark curiosity in physics",
+      colour: "yellow",
+      image_1_path: "static/content/hero-images/0012.jpg",
+      image_1_alt: "A teacher sitting with a student helping them with a maths problem",
+      image_2_path: "static/content/hero-images/0007.jpg",
+      image_2_alt: "A maths teacher standing at the front of a full classroom delivering a lesson",
```

Comment:
This alt text doesn't correspond to the image? We shouldn't be writing alt text directly on a page anyway - it should be pulled from the central images.yml file automatically?